src/simulation/Benchmark.h

```diff
@@ -0,0 +1,178 @@
+#pragma once
+
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "main/Application.h"
+#include "medida/counter.h"
+#include "medida/metrics_registry.h"
+#include "medida/reporting/json_reporter.h"
+#include "medida/timer.h"
+#include "simulation/LoadGenerator.h"
+#include "util/Timer.h"
+#include <chrono>
+#include <memory>
+#include <vector>
+
+namespace stellar
+{
+
+class TxSampler;
+
+class Benchmark
```

Comment:
there are a lot of classes in this file
I propose creating directory benchmark with several files in it (one big class per file)